Fb App Events ios device mode updates (#853)
rudderlabs/rudderstack-docs: docs/destinations/streaming-destinations/facebook-app-events.mdx

```
</span>
</li>
</ol>
      </TabPanel>
    </TabPanels>
</Tabs>

## Identify

<div class="warningBlock">

> For App Events, RudderStack supports the <code class="inline-code">identify</code> calls only in the mobile device mode.

Reviewer: I thinks it would be more appropriate if we mention `Facebook`:
```suggestion
For Facebook App Events, RudderStack supports the <code class="inline-code">identify</code> calls only in the mobile device mode.
```

## Track

> The <Link to="/event-spec/standard-events/track">`track`</Link> call is logged to Facebook using the `logEvent` method of the `AppEventsLogger` class. RudderStack uses the same `eventName` as you have passed to the `track` method. Along with that, it will pass all the `properties` after converting them into an accepted format.

Reviewer 2: ```suggestion
RudderStack logs the <Link to="/event-spec/standard-events/track">`track`</Link> call to Facebook using the `logEvent` method of the `AppEventsLogger` class. It uses the same `eventName` as you have passed to the `track` method along with all the `properties`, after converting them into an accepted format.
```

```

<div class="infoBlock">
When <code class="inline-code">revenue</code> and <code class="inline-code">currency</code> are present in the event properties of any <code class="inline-code">track</code> call, RudderStack makes a <code class="inline-code">Purchase</code> call to Facebook using its <code class="inline-code">logPurchase</code> API along with the normal <code class="inline-code">track</code> call using the <code class="inline-code">logEvent</code> API.

If <code class="inline-code">currency</code> is absent in the event properties, RudderStack sets the default value to <code class="inline-code">USD</code>.
</div>

### Supported mappings for iOS v2

> This section lists some `track` event and property mappings which are **applicable only** for <Link to="/sources/event-streams/sdks/rudderstack-ios-sdk/ios-v2/">iOS v2</Link> device mode.

Reviewer 2: ```suggestion
This section lists some `track` event and property mappings which are **applicable only** when sending events via the <Link to="/sources/event-streams/sdks/rudderstack-ios-sdk/ios-v2/">iOS v2</Link> device mode.
```

  </TabPanels>
</Tabs>

<div class="infoBlock">

> Starting from iOS version 14.5, you need to get the device consent to share data with Facebook or not by setting the  <code class="inline-code">isAdvertiserTrackingEnabled</code> property. Refer to Facebook's <a href="https://developers.facebook.com/docs/app-events/getting-started-app-events-ios#get-device-consent">Get device consent</a> documentation for more information.

Reviewer: Just a minor suggestion, if looks appropriate then include it:
```suggestion
Starting from iOS version 14.5, you need to get the device consent to share data with Facebook, by setting the  <code class="inline-code">isAdvertiserTrackingEnabled</code> property. Refer to Facebook's <a href="https://developers.facebook.com/docs/app-events/getting-started-app-events-ios#get-device-consent">Get device consent</a> documentation for more information.
```

Reviewer 2: Made a minor tweak by removing the comma, looks good otherwise. thanks for your suggestion!
